kamelets/mongodb-source.kamelet.yaml

```diff
@@ -71,7 +71,7 @@ spec:
         type: string
       persistentTailTracking:
         title: MongoDB Persistent Tail Tracking
-        description: Specifies to enable persistent tail tracking, which is a mechanism to keep track of the last consumed data across system restarts. The next time the system is up, the endpoint recovers the cursor from the point where it last stopped consuimg data.
+        description: Specifies to enable persistent tail tracking, which is a mechanism to keep track of the last consumed data across system restarts. The next time the system is up, the endpoint recovers the cursor from the point where it last stopped consuimg data. This option will only work on "capped" collections.
```

Comment:
I think it may be good to avoid using double quotes / single quotes in the description as tooling may have a problem with that